pages/apis/rest_api.md

```diff
@@ -81,6 +87,34 @@ You can set the page using the following query string parameters:
 </tbody>
 </table>
 
+### Cursor pagination
+
+Some endpoints are paginated using cursors, with the `Link` header containing zero or more of `next` and `prev` only.
+
+For example, the following request for all builds will return a response with a cursor link for `next`:
+
+```bash
+curl -i "https://api.buildkite.com/v2/builds"
+```
+
+```
+HTTP/1.1 200 OK
+...
+Link <https://api.buildkite.com/v2/builds?after=eyJjcmVhdGVkX2F0IjoiMjAyMy0wOC0yMyAwMjoyOTo1My4yNjA5MzMwMDAgVVRDIiwidXVpZCI6IjAxOGEyNTk3LTY0NzAtNGRjZC05MjIxLWY3OGQ2ZmFiZDkwNiJ9>; rel="next"
+```
+
+You can page through the records using the following query string parameters:
+
+<table>
+<tbody>
+  <tr><th><code>after</code></th><td>Return a page of results after this cursor (for paginating forward).<p>Specify one of <code>after</code> or <code>before</code> only.</p></td></tr>
+  <tr><th><code>before</code></th><td>Return a page of results before this cursor (for paginating backward).<p>Specify one of <code>after</code> or <code>before</code> only.</p></td></tr>
+  <tr><th><code>per_page</code></th><td>How many results to return per-page<p class="Docs__api-param-eg"><em>Default:</em> <code>30</code></p><p class="Docs__api-param-eg"><em>Maximum:</em> <code>100</code></p></td></tr>
+</tbody>
+</table>
+
+We recommend you do not need construct cursor-paginated URLs manually. Instead, use the `next` and `prev` links in the `Link` header.
```

Comment:
Ah, a fix I put in my last suggestion that I didn't call out was the typo in "need construct." I assume it's a mix of "you do not need to construct" and "we recommend you do not construct." I'd go with:

```suggestion
We recommend you do not construct cursor-paginated URLs manually. Instead, use the `next` and `prev` links in the `Link` header.
```

Reply:
Oh yes, thank you! Will sort.